css-view-transitions-1/Overview.bs

```diff
@@ -770,25 +770,22 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 ## View Transition Painting Order ## {#view-transition-stacking-layer}
 
 	This specification introduces a new stacking layer, the [=view transition layer=],
-	to the <a href="https://www.w3.org/TR/CSS22/zindex.html">Elaborate Description of Stacking Contexts</a>.
-
-	The ''::view-transition'' pseudo-element generates a new stacking context
-	called the <dfn>view transition layer</dfn>
-	with the following characteristics:
-
-	- Its parent [=stacking context=] is the root stacking context.
-
-	- If the ''::view-transition'' pseudo-element exists,
-		a new [=stacking context=] is created for the [=document element=] and the [=Document/top layer=].
-		The [=view transition layer=] is a sibling of this stacking context.
-
-	- The [=view transition layer=] paints after the stacking context for the [=document element=] and [=Document/top layer=].
-		This includes the filters and effects that are applied to the [=document element=].
+	to the end of the painting order established in
+	<a href="https://www.w3.org/TR/CSS22/zindex.html">CSS2&sect;E Elaborate Description of Stacking Contexts</a>. [[!CSS2]]
+
+	The ''::view-transition'' pseudo-element generates a new stacking context,
+	called the <dfn>view transition layer</dfn>,
+	which paints after all other content of the document
```

Comment:
One point which I want to make sure is obvious is that the root and top layer elements don't paint during the regular DOM if they're "captured in a view transition", a.k.a, computed value of view-transition-name on the root element is not none.

The spec text here is supposed to cover that: https://drafts.csswg.org/css-view-transitions-1/#setup-transition-pseudo-elements-algorithm:~:text=The%20new%20element,until%20new%20exists. If you have any better ideas to include that here (even in a normative note), would be good!